### DISCUSSION TITLE
Continuing Incomplete Jobs

### DISCUSSION THREAD

**Discussion by omorrison1** [MarsalekGroup/aml#3] [Q&A] [answered]
Hello,

I have been trying to run QbC jobs with AML, but have found that long jobs stop halfway through with no errors or logs explaining why.

I have yet to find the cause of this issue.

I was wondering if there is some way that I can continue an incomplete QbC job.

For example, the most recent job stopped at iteration 8 of 10. Is there some way that I can rerun the QbC for just the last 3 iterations?

Thanks,
Oliver

**Comment by OndrejMarsalek** [comment, marked as answer]
Hello, this is still a little rough around the edges, but there is a way to restart. The file `results.shelf` (can be renamed using `fn_results` if desired) is updated each iteration. If you provide it to a new run using `fn_restart` when constructing the `QbC` object, the process will continue from there.

Unfortunately, I do not know what could cause the run to stop with no error messages.
